refresh/fromswagger/swaggerize.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+var debug = require('debug')('refresh/fromSwagger/swaggerize');
```

Comment:
IIRC the convention with the debug library is to separate submodules with `:` not `/`.